frontend/package.json

```diff
@@ -38,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "datejs": "^1.0.0-rc3"
```

Comment:
datejs be in "dependencies", not "devDependencies"

Reply:
okay, has been changed to be in dependencies